spring-kafka-docs/src/main/java/org/springframework/kafka/jdocs/topics/Config.java

```diff
@@ -67,9 +67,9 @@ public NewTopic topic2() {
     @Bean
     public NewTopic topic3() {
         return TopicBuilder.name("thing3")
-                .assignReplicas(0, Arrays.asList(0, 1))
-                .assignReplicas(1, Arrays.asList(1, 2))
-                .assignReplicas(2, Arrays.asList(2, 0))
+                .assignReplicas(0, List.of(0, 1))
+                .assignReplicas(1, List.of(1, 2))
+                .assignReplicas(2, List.of(2, 0))
```

Comment:
not necessary, but we have used Java 17, why not use the handy collection constant feature introduced since Java 9?